linode-launch.py

```diff
@@ -48,85 +49,136 @@ def releasing(semaphore):
 create_semaphore = BoundedSemaphore(15)
 config_semaphore = BoundedSemaphore(8)
 def do_create(client, running, datacenter, plans, distribution, kernel, machine, i):
-    plan = filter(lambda p: p[u'LABEL'].lower().find(str(machine['plan']).lower()) >= 0, plans)[0]
+    
+    for p in plans:
+        str_plan_label = p.label
+        str_reduced_plan_label = str_plan_label.split(" ")[1]
+        if str(machine['plan']).lower() == str_reduced_plan_label.lower():
+            plan = p
+            logging.info("plan: %s" % plan)
+            
     name =  u'{0}-{1:03d}'.format(machine['prefix'], i)
     label =  u'{0}-{1}'.format(GROUP, name)
     logging.info("%s", "creating {}".format(label))
 
-    active = filter(lambda x: x[u'LABEL'] == label and x[u'LPM_DISPLAYGROUP'] == GROUP, running)
-    assert(len(active) <= 1)
+    active = ""
+    for node in running:
+        str_label = node.label
+        str_group = GROUP
+        if str_label == label and str_group == GROUP:
+            active = node
+            logging.info("active: %s" % active)
+
     if active:
-        logging.info("%s", "linode {} already exists: {}".format(label, active[0][u'LINODEID']))
-        node = {u'LinodeID': active[0][u'LINODEID']}
+        logging.info("%s", "linode {} already exists: {}".format(label, active.id))
+        node = client.load(active, active.id)
     else:
         with releasing(create_semaphore):
-            node = client.linode_create(DatacenterID = datacenter, PlanID = plan[u'PLANID'], PaymentTerm = 1)
-            client.linode_update(LinodeID = node[u'LinodeID'], Label = label, lpm_displayGroup = GROUP, watchdog = 1, Alert_cpu_enabled = 0)
-            logging.info("created %s: %s", label, node[u'LinodeID'])
+            node = client.linode.instance_create(ltype = plan.id, 
+                                               region = datacenter,
+                                               label = label,
+                                               group = GROUP,
+                                               tags = [GROUP])
+            
+            logging.info("created %s: %s", label, node.id)
 
     with releasing(config_semaphore):
         try:
-            client.linode_ip_addprivate(LinodeID = node[u'LinodeID'])
```

Comment:
Since we cannot configure the alerts or watchdog in `instance_create`, please do that here in a call to `linode.update`. We do not want any alerts and we **do** want the watchdog enabled.